compiler/il/OMRILOps.hpp

```diff
@@ -866,13 +866,12 @@ class ILOpCode
       return TR::BadILOp;
       }
 
+//TO DO: remove this function as unsigned shfit opcodes have been deprecated
```

Comment:
As the comment suggests, I think this function can just be removed.

Reply:
May I remove the function in another PR as this is outside of scope of this PR. I will remove the function along with all other similar functions as listed in https://github.com/eclipse/omr/issues/2657#issuecomment-508139949.

Reply:
Sounds like you may have wanted to write in the code then:

```
// TODO (#2657): ...
```

Please try to use the canonical name "TODO" as is the convention in the rest of the codebase. For example someone searching for todo's to fix is unlikely to search for "to do" (with a space) as such convention is not used anywhere in the codebase currently.

Reply:
Fixed in the new force-push.